lang/rust/pkg/config.toml.in

```diff
@@ -0,0 +1,35 @@
+changelog-seen = 2
+
+[build]
+# cargo = "PREFIX/bin/cargo"
+docs = false
+# Triggers the build of tools. Cargo, in our case.
+extended = true
+# Mainly needed for cross-language PGO in Firefox.
+profiler = true
+# rustc = "PREFIX/bin/rustc"
+# TODO i686 (needed for multilib)
+target = ["x86_64-unknown-linux-gnu"]
+tools = ["cargo", "clippy", "rustfmt"]
+vendor = true
+
+[install]
+libdir = "LIBDIR"
+prefix = "PREFIX"
+
+[llvm]
+link-shared = true
+optimize = true
+targets = "X86"
+
+[rust]
+channel = "stable"
+description = "SerpentOS Linux"
```

Comment:
I think our branding is "Serpent OS" alone (with a whitespace in between), plus I think we're not allowed to use "Linux" since it's a trademark.

Reply:
Agreed